docs/backend/widgets.md

```diff
@@ -12,15 +12,688 @@ myst:
 # Widgets
 
 ```{seealso}
-See the chapter {ref}`training:dexterity-reference-label` from the Mastering Plone 6 Training.
+See the chapter {doc}`training:mastering-plone/dexterity_reference` from the Mastering Plone 6 Training.
 ```
 
-```{todo}
-Contribute to this documentation!
-See issue [Backend > Widgets needs content](https://github.com/plone/documentation/issues/1307).
+Widgets render HTML code for input and parse HTTP `post` request input in Plone.
+
+Plone stores widgets as the `widgets` attribute of a form.
+Plone presents widgets as an ordered dict-like `Widgets` class.
+
+Widgets are only available after you call the form's `update()` and `updateWidgets()` methods.
+`updateWidgets()` binds widgets to the form's context.
+{doc}`vocabularies` defined by their names are resolved at this point.
+
+The Zope publisher will also mangle widget names based on what kind of input the widget accepts.
+When an HTTP `post` request comes in, Zope publisher automatically converts `<select>` dropdowns to lists.
+
+
+## Widget reference
+
+You can find the default widgets in the browser package in `z3c.form`.
+The [`z3c.form` documentation](https://z3cform.readthedocs.io/en/latest/widgets/index.html) lists all the default widgets and shows the HTML output of each.
+
+You can find more widgets in the [`plone.app.z3cform`](https://github.com/plone/plone.app.z3cform/tree/master) package.
+
+The main widgets are:
+
+-   Checkbox Widget
+-   Radio Widget
+-   Text Widget
+-   TextArea Widget
+-   TextLines Widget
+-   Password Widget
+-   Select Widget
+-   Ordered-Select Widget
+-   File Widget
+-   File Testing Widget
+-   Image Widget
+-   Multi Widget
+-   Object Widget
+-   Button Widget
+-   Submit Widget
+-   Date Widget
+-   DateTime Widget
+-   Time Widget
+-   DateTime Picker
+
+## Changing a field's widget
+
+You can change a the widget that you use for a field in several ways.
+This section describes these methods.
+
+
+### `plone.autoform` `widget` directive
+
+`plone.autoform` builds custom `z3c.form` forms based on a model (schema) of fields, and their widgets and options.
+This model is defined as a `zope.schema` based schema, but extra hints can be supplied to control aspects of form display not normally specified in a Zope schema.
+
+By default, `z3c.form` picks a widget based on the type of your field.
+You can change the widget using the `widget` directive if you want users to enter or view data in a different format.
+For example, you can change the widget for the `human` ``boolean`` field to use "yes" and "no" radio buttons instead of its default checkbox:
```

Comment:
"boolean" is not an inline literal.

Side note, we use only single backticks for inline literals in MyST or Markdown.

```suggestion
For example, you can change the widget for the `human` boolean field to use "yes" and "no" radio buttons instead of its default checkbox:
```